src/benchmark/operators/join_benchmark.cpp

```diff
@@ -1,87 +1,84 @@
 #include <memory>
 
-#include "../benchmark_join_fixture.hpp"
 #include "../table_generator.hpp"
 #include "benchmark/benchmark.h"
 #include "operators/join_hash.hpp"
 #include "operators/join_index.hpp"
 #include "operators/join_nested_loop.hpp"
 #include "operators/join_sort_merge.hpp"
 #include "operators/table_wrapper.hpp"
+#include "storage/chunk.hpp"
+#include "storage/index/adaptive_radix_tree/adaptive_radix_tree_index.hpp"
+#include "storage/storage_manager.hpp"
 
 namespace opossum {
 
-BENCHMARK_DEFINE_F(BenchmarkJoinFixture, BM_JoinHash)(benchmark::State& state) {
-  clear_cache();
-
-  auto warm_up =
-      std::make_shared<JoinHash>(_tw_small_uni1, _tw_small_uni2, JoinMode::Inner,
-                                 std::pair<ColumnID, ColumnID>{ColumnID{0}, ColumnID{0}}, PredicateCondition::Equals);
-  warm_up->execute();
-  while (state.KeepRunning()) {
-    auto table_scan =
-        std::make_shared<JoinHash>(_tw_small_uni1, _tw_small_uni2, JoinMode::Inner,
-                                   std::pair<ColumnID, ColumnID>(ColumnID{0}, ColumnID{0}), PredicateCondition::Equals);
-    table_scan->execute();
+void clear_cache() {
+  std::vector<int> clear = std::vector<int>();
+  clear.resize(500 * 1000 * 1000, 42);
+  for (uint i = 0; i < clear.size(); i++) {
+    clear[i] += 1;
   }
+  clear.resize(0);
 }
 
-BENCHMARK_DEFINE_F(BenchmarkJoinFixture, BM_JoinSortMerge)(benchmark::State& state) {
-  clear_cache();
+std::shared_ptr<TableWrapper> generate_table(const size_t number_of_rows, const size_t number_of_chunks,
+                                             bool numa_distribute_chunks = false) {
```

Comment:
This looks like the PR is not based on the master? :/

Reply:
It is, I had the `numa_distribute_chunks` commented out when passing to the TableGenerator after the PR being reverted (with the thought that once #835 is in, we can use it again), but since this is going to change anyway, I removed it completely now and am about to push ;)